relay-server/src/actors/store.rs

```diff
@@ -374,8 +385,24 @@ impl StoreForwarder {
     }
 
     fn send_metric_message(&self, message: MetricKafkaMessage) -> Result<(), StoreError> {
+        let topic = match message.name.parse() {
+            Ok(MetricMri { namespace, .. }) if namespace == "transactions" => {
+                KafkaTopic::MetricsTransactions
+            }
+            Ok(MetricMri { namespace, .. }) if namespace == "sessions" => {
+                KafkaTopic::MetricsSessions
+            }
+            _ => {
+                relay_log::configure_scope(|scope| {
+                    scope.set_extra("metric_message.name", message.name.into());
+                });
+                relay_log::error!("Dropping unknown metric usecase");
```

Comment:
That should ideally happen earlier in the pipeline.

Reply:
Do we want to allow custom namespaces or not? If it happens earlier in the pipeline I'd assume you want to apply that logic unconditionally, regardless of whether it's a processing relay or not.